### PR TITLE
[IP-614] Chore/write tests

### PR DESCRIPTION
**Pull request by ynnadkrap**
@viewthespace/toolbox

### CODE REVIEW

File: spec/model_renamer/migration_generator_spec.rb [hunk 1]
```diff
@@ -0,0 +1,61 @@
+describe MigrationGenerator do
+  after(:all) do
+    FileUtils.rm_rf('./db')
+  end
```
Comment (ynnadkrap) [inline spec/model_renamer/migration_generator_spec.rb]: Open to suggestions here. Got some strange errors when I tried to use MemFs here, so I let rails do its thing and create an actual migration file and then delete it at the end.

**Comment by ynnadkrap** [comment]
Should we set this up to run on circle ci?

**Comment by kbaum** [comment]
great idea!

File: lib/model_renamer/rename.rb [hunk 2]
```diff
@@ -41,12 +47,7 @@ def all_filepaths
   end
 
   def acceptable_filetype? path
-    [
-      'js', 'coffee', 'hamlc', 'skim', 'erb',
-      'sass', 'scss', 'css', 'rb', 'slim',
-      'haml', 'rabl', 'html', 'txt', 'feature',
-      'rake', 'json', 'sh', 'yaml', 'sql'
-    ].include? path.split('.').last
+    ACCEPTABLE_FILE_TYPES.include? path.split('.').last
```
Comment (kbaum) [inline lib/model_renamer/rename.rb:51]: Well done!

File: spec/model_renamer/migration_generator_spec.rb (hunk 1 above)
Comment (kbaum) [reply spec/model_renamer/migration_generator_spec.rb]: Interesting.. Should this just be an `after(:each)`?

**Review by kbaum — APPROVED** [review]

File: spec/model_renamer/migration_generator_spec.rb [hunk 3]
```diff
@@ -0,0 +1,61 @@
+describe MigrationGenerator do
+  after do
```
Comment (kbaum) [inline spec/model_renamer/migration_generator_spec.rb:2]: @ynnadkrap - changed to just plain `after do` and seems to work.

**Pull request merged**